js/intro.js

```diff
@@ -0,0 +1,320 @@
+    type: 'p',
+    classList: ['input-label-dark'],
+  });
+  introduction.innerText = 'Introduction';
+  const renderIntro = createElement({
+    type: 'p',
+    classList: ['user-input', 'input-big'],
+    id: 'renderIntro',
+  });
+  renderIntro.innerHTML = users.introduction;
+  container.appendChild(introduction);
+  container.appendChild(renderIntro);
+
+  const skills = createElement({ type: 'p', classList: ['input-label-dark'] });
+  skills.innerText = 'Skills';
+  const renderSkills = createElement({
+    type: 'p',
+    classList: ['user-input', 'input-regular'],
+    id: 'renderSkills',
+  });
+  renderSkills.innerHTML = users.skills;
+  container.appendChild(skills);
+  container.appendChild(renderSkills);
+
+  const institution = createElement({
+    type: 'p',
+    classList: ['input-label-dark'],
+  });
+  institution.innerText = 'Institution';
+  const renderInstitution = createElement({
+    type: 'p',
+    classList: ['user-input', 'input-regular'],
+    id: 'renderSkills',
+  });
+  renderInstitution.innerHTML = users.institution;
+  container.appendChild(institution);
+  container.appendChild(renderInstitution);
+
+  const forFun = createElement({ type: 'p', classList: ['input-label-dark'] });
+  forFun.innerText = 'What you do for fun';
+  const renderForFun = createElement({
+    type: 'p',
+    classList: ['user-input', 'input-big'],
+    id: 'renderForFun',
+  });
+  renderForFun.innerHTML = users.forFun;
+  container.appendChild(forFun);
+  container.appendChild(renderForFun);
+
+  const funFact = createElement({ type: 'p', classList: ['input-label-dark'] });
+  funFact.innerText = 'A fun fact about you';
+  const renderFunFact = createElement({
+    type: 'p',
+    classList: ['user-input', 'input-big'],
+    id: 'renderFunFact',
+  });
+  renderFunFact.innerHTML = users.funFact;
+  container.appendChild(funFact);
+  container.appendChild(renderFunFact);
+
+  const whyRds = createElement({ type: 'p', classList: ['input-label-dark'] });
+  whyRds.innerText = 'Why do you want to join Real Dev Squad?';
+  const renderWhyRds = createElement({
+    type: 'p',
+    classList: ['user-input', 'input-big'],
+    id: 'renderWhyRds',
+  });
+  renderWhyRds.innerHTML = users.whyRds;
+  container.appendChild(whyRds);
+  container.appendChild(renderWhyRds);
+
+  const heardAbout = createElement({
+    type: 'p',
+    classList: ['input-label-dark'],
+  });
+  heardAbout.innerText = 'How did you hear about us';
+  const renderHeardAbout = createElement({
+    type: 'p',
+    classList: ['user-input', 'input-regular'],
+    id: 'renderHeardAbout',
+  });
+  renderHeardAbout.innerText = users.foundFrom;
+  container.appendChild(heardAbout);
+  container.appendChild(renderHeardAbout);
+  loading.classList.add('hidden');
+  document.querySelector('.intro-main').appendChild(renderIntroPage);
+}
+
+//making userSavedData object from API
+async function showSavedDetails() {
+  try {
+    const userId = getUserId();
```

Comment:
Do check the above comment related to **URLSearchParams** and make changes here.